trie/triedb/pathdb/disklayer.go

```diff
@@ -183,7 +183,7 @@ func (dl *diskLayer) commit(bottom *diffLayer, force bool) (*diskLayer, error) {
 		}
 	}
 	// Mark the diskLayer as stale before applying any mutations on top.
-	dl.stale = true
+	//dl.stale = true
```

Comment:
Why comment this line? If necessary, add some comment please.

Reply:
revert